### PR TITLE
Update Color.md

### PR DESCRIPTION
**Pull request by Arashvscode**
Activate the Java magic color in the readme and give a more beautiful front!

### CODE REVIEW

File: docs/theming/Color.md [hunk 1]
```diff
@@ -403,7 +403,7 @@ package:
 
 In your application class or activity/fragment/view, call:
 
-```
+``` java
```
Comment (afohrman) [inline docs/theming/Color.md:406]: Nit: extra space before "java".

Apply throughout.

**Review by afohrman — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good except for extra spacing, see comment within.

**Comment by dsn5ft** [comment]
Sorry to hear that, I'll manually apply these changes and close the PR then.

**Comment by Arashvscode** [comment]
> 

ok bro

**Pull request closed without merging**